### PR TITLE
add allure properties and fix issue links (fixes #44)

### LINKED ISSUE #44: Use svg executor logos
* Jenkins - http://jenkins.mirror.isppower.de/art/jenkins-logo/headshot.svg
* Teamcity - https://resources.jetbrains.com/assets/products/teamcity/teamcity_logos.zip

### PR DESCRIPTION
**Pull request by ehborisov**

### CODE REVIEW

File: allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java [hunk 1]
```diff
@@ -71,19 +74,36 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(Allure1Plugin.class);
     private static final String UNKNOWN = "unknown";
     private static final String MD_5 = "md5";
+    private static final String ISSUE_URL_PROPERTY = "allure.issues.tracker.pattern";
+    private static final String TMS_LINK_PROPERTY = "allure.tests.management.pattern";
 
     private final ObjectMapper mapper = new ObjectMapper();
+    private Properties allureProperties;
```
Comment (baev) [inline allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java]: remove the field

File: allure-generator/src/test/resources/allure1/allure.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+allure.issues.tracker.pattern=http://bugtracker.url/%s
```
Comment (baev) [inline allure-generator/src/test/resources/allure1/allure.properties]: use example.org instead

File: allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java [hunk 3]
```diff
@@ -71,19 +72,33 @@
...
         getStreamOfAllure1Results(resultsDirectory).forEach(testSuite -> testSuite.getTestCases()
                 .forEach(testCase -> convert(context.getValue(), resultsDirectory, visitor, testSuite, testCase))
         );
     }
 
+    private void loadAllureProperties(final Path resultsDirectory) {
+        final Path propertiesFile = resultsDirectory.resolve("allure.properties");
+        if (Files.exists(propertiesFile)) {
+            try (final FileInputStream propFile = new FileInputStream(propertiesFile.toFile())) {
+                System.getProperties().load(propFile);
```
Comment (baev) [inline allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java]: The problem is that system properties should override properties from file. You can create load properties from file to separate object, and then pass it to methods

File: allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java [hunk 4]
```diff
@@ -71,25 +74,44 @@
...
+        if (Files.exists(propertiesFile)) {
+            try (final FileInputStream propFile = new FileInputStream(propertiesFile.toFile())) {
+                final Properties properties = new Properties();
+                properties.load(propFile);
+                return properties;
+            } catch (IOException e) {
+                LOGGER.error("Error while reading allure.properties file: %s", e.getMessage());
+            }
+        }
+        return new Properties();
```
Comment (eroshenkoam) [inline allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java]: Properties properties = new Properties();
if (file exist) {
...
properties.load()
...
} 
return properties;

**Pull request merged**